packages/appium/lib/main.js

```diff
@@ -170,6 +172,19 @@ function areServerCommandArgs(args) {
 async function init(args) {
   const appiumHome = args?.appiumHome ?? (await resolveAppiumHome());
 
+  // this is here so extensions installed in e.g., `~/.appium` can find the peer dependency
+  // of `appium`, which lives wherever it lives (see `rootDir`).
+  if (!(await hasAppiumDependency(appiumHome))) {
+    try {
+      await fs.mkdirp(path.join(appiumHome, 'node_modules'));
+      await fs.symlink(rootDir, path.join(appiumHome, 'node_modules', 'appium'), 'junction');
```

Comment:
woah what is this `junction` business?

Reply:
oh sorry i missed the `)` and misread `junction` as the last argument of `path.join`, not an arg of `fs.symlink`